install/kubernetes/cilium/charts/agent/templates/svc.yaml

```diff
@@ -17,3 +17,43 @@ spec:
   selector:
     k8s-app: cilium
 {{- end }}
+{{- if .Values.global.hubble.ui.enabled }}
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: hubble-grpc
+  namespace: {{ .Release.Namespace }}
+  labels:
+    k8s-app: cilium
```

Comment:
Shouldn't this be `k8s-app: hubble`?

Reply:
No. The service is running inside cilium-agent.

Reply:
@gandro this is not the service selector, these are the labels of the service.

Reply:
Oh, didn't realize that! I agree that this can be to `k8s-app: hubble` @soumynathan